userguide/payment/includes/payment-methods-flows.adoc

```diff
@@ -88,4 +88,13 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/paym
 
 . Kill Bill invokes the `PaymentPlugin#addPaymentMethod` method.
 
-. The `PaymentPlugin#addPaymentMethod` performs a gateway specific call to tokenize the payment details and obtains a token. This is saved within the Kill Bill database.
\ No newline at end of file
+. The `PaymentPlugin#addPaymentMethod` performs a gateway specific call to tokenize the payment details and obtains a token. This is saved within the Kill Bill database.
+
+
+=== Plugin Name
+
+Along with the user's payment method details, a client application also needs to send the plugin name to the addPaymentMethod method. This helps Kill Bill to route the payment processing to the appropriate plugin
```

Comment:
Nit: missing trailing dot.

Reply:
Fixed, I've removed this section as its not required, plugin names are explained in a separate document